src/Features/Core/Portable/SolutionCrawler/AbstractDocumentDifferenceService.cs

```diff
@@ -15,23 +15,23 @@ namespace Microsoft.CodeAnalysis.SolutionCrawler;
 
 internal abstract class AbstractDocumentDifferenceService : IDocumentDifferenceService
 {
-    public async Task<DocumentDifferenceResult?> GetDifferenceAsync(Document oldDocument, Document newDocument, CancellationToken cancellationToken)
+    public async Task<SyntaxNode?> GetChangedMemberAsync(Document oldDocument, Document newDocument, CancellationToken cancellationToken)
```

Comment:
see comment in https://github.com/dotnet/roslyn/pull/72147/files#r1516807099 for explanation.